packages/core/strapi/lib/services/entity-validator/validators.js

```diff
@@ -113,11 +155,11 @@ module.exports = {
   uid: uidValidator,
   json: () => yup.mixed(),
   integer: integerValidator,
-  biginteger: () => yup.mixed(),
+  biginteger: composeValidators(() => yup.mixed(), addUniqueValidator),
```

Comment:
the yup.mixed() isn't necessary if you use composeValidators as it's the default validator anyway